Assets/MRTK/Tests/PlayModeTests/BoundingBoxTests.cs

```diff
@@ -109,6 +109,67 @@ public IEnumerator BBoxOverride()
             yield return null;
         }
 
+        /// <summary>
+        /// Tests to see that the handlers grow in size when a pointer is near them
+        /// </summary>
+        [UnityTest]
+        public IEnumerator BBoxHandlerUI()
+        {
+            const int numSteps = 2;
+            var bbox = InstantiateSceneAndDefaultBbox();
+            bbox.ShowRotationHandleForX = true;
+            bbox.ShowRotationHandleForY = true;
+            bbox.ShowRotationHandleForZ = true;
+            bbox.ShowScaleHandles = true;
+            bbox.ProximityEffectActive = true;
+            bbox.ScaleHandleSize = 0.1f;
+            bbox.RotationHandleSize = 0.1f;
+            bbox.FarScale = 1.0f;
+            bbox.MediumScale = 1.5f;
+            bbox.CloseScale = 1.5f;
+            yield return null;
+            var bounds = bbox.GetComponent<BoxCollider>().bounds;
+            Assert.AreEqual(new Vector3(0, 0, 1.5f), bounds.center);
+            Assert.AreEqual(new Vector3(.5f, .5f, .5f), bounds.size);
+
+            var inputSimulationService = PlayModeTestUtilities.GetInputSimulationService();
+
+            // Definining the edge and corner handlers that will be used
+            var originalCornerHandlerScale = bbox.ScaleCornerVisuals[0].transform.localScale;
```

Comment:
getting a scale handle can be achieved by calling for example: transform.Find("rigRoot/corner_0") and it's visual by calling transform.Find("rigRoot/corner_0/visualsScale/visuals") on the boundingbox gameobject
if you want to have the front right corner that would be corner_3 . hope that helps :)